src/components/Toast/Toast.tsx

```diff
@@ -93,34 +93,34 @@ function useToastStyles({type}: {type: ToastType}) {
       default: {
         backgroundColor: select(t.name, {
           light: t.atoms.bg_contrast_25.backgroundColor,
-          dim: t.atoms.bg_contrast_100.backgroundColor,
-          dark: t.atoms.bg_contrast_100.backgroundColor,
+          dim: t.atoms.bg_contrast_25.backgroundColor,
+          dark: t.atoms.bg_contrast_25.backgroundColor,
         }),
         borderColor: select(t.name, {
-          light: t.atoms.border_contrast_low.borderColor,
+          light: t.atoms.border_contrast_high.borderColor,
           dim: t.atoms.border_contrast_high.borderColor,
           dark: t.atoms.border_contrast_high.borderColor,
         }),
         iconColor: select(t.name, {
-          light: t.atoms.text_contrast_medium.color,
-          dim: t.atoms.text_contrast_medium.color,
-          dark: t.atoms.text_contrast_medium.color,
+          light: t.atoms.text.color,
+          dim: t.atoms.text.color,
+          dark: t.atoms.text.color,
         }),
         textColor: select(t.name, {
-          light: t.atoms.text_contrast_medium.color,
-          dim: t.atoms.text_contrast_medium.color,
-          dark: t.atoms.text_contrast_medium.color,
+          light: t.atoms.text.color,
+          dim: t.atoms.text.color,
+          dark: t.atoms.text.color,
         }),
       },
       success: {
         backgroundColor: select(t.name, {
-          light: t.palette.primary_100,
-          dim: t.palette.primary_100,
+          light: t.palette.primary_25,
+          dim: t.palette.primary_25,
           dark: t.palette.primary_50,
```

Comment:
Realize this isn't marked as ready, but just wanted to note that I think a few values don't match Figma, including this one, which I think should be `primary_25` as well.

If all these values are the same for all themes, we can get rid of all the `select` calls here 👍